src/components/PromotionCard/PromotionCard.module.css

```diff
@@ -3,10 +3,19 @@
   flex-direction: column;
   width: 100%;
   max-width: 340px;
+  overflow: hidden;
   text-align: center;
   box-shadow: 2px 2px 10px rgba(0, 0, 0, .25), -2px -2px 10px rgba(0, 0, 0, .25);
```

Comment:
`transition: transform .5s ease-in, opacity .8s ease-in, box-shadow .3s ease-in`